src/main/java/com/feed_the_beast/ftbu/world/FTBUUniverseData.java

```diff
@@ -228,6 +232,11 @@ public static boolean claimChunk(IForgePlayer player, ChunkDimPos pos)
 
     public static boolean unclaimChunk(IForgePlayer player, ChunkDimPos pos)
     {
+        if (FTBUConfigWorld.LOCKED_IN_CLAIMED_CHUNKS.getBoolean()
```

Comment:
Yes, in `ClaimedChunkStorage` there is a check that if chunk is unclaimed, then with this flag on they can't interact with blocks.